.buildkite/scripts/steps/functional/security_serverless.sh

```diff
@@ -8,6 +8,9 @@ source .buildkite/scripts/steps/functional/common_cypress.sh
 export JOB=kibana-serverless-security-cypress
 export KIBANA_INSTALL_DIR=${KIBANA_BUILD_LOCATION}
 
+echo "$KIBANA_DOCKER_PASSWORD" | docker login -u "$KIBANA_DOCKER_USERNAME" --password-stdin docker.elastic.co
```

Comment:
Maybe since the login<->logout-trap goes together, and is repeated in two places, this could live in the common utils as a function? (.buildkite/scripts/common/util.sh)